apph-front/src/services/TableService.tsx

```diff
@@ -0,0 +1,37 @@
+import Cookies from 'universal-cookie';
+import Server from './Server';
+
+const cookies = new Cookies();
+
+class TableService {
+  static getData(
+    setData: (val: any) => void,
+    handleSuccess: () => void,
+    handleError: (errorMessage: string) => void
+  ) {
+    const URL = `/photo/infos`;
+    const userInfos = cookies.get('user');
+    const requestOptions = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        token: userInfos?.token
+      }
+    };
+    const successFunction = (val: any) => {
+      let i = 1;
+      val = JSON.parse(val);
+      for (let elt of val) {
```

Comment:
```suggestion
      for (const elt of val) {
```

j'ai une erreur eslint qui me dit d'utiliser const ici, y'a aussi d'autre warning eslint dans ce fichier